agent/app/agent_capability.go

```diff
@@ -185,6 +186,9 @@ func (agent *ecsAgent) capabilities() ([]*ecs.Attribute, error) {
 	// support aws router capabilities for log driver router
 	capabilities = agent.appendFirelensLoggingDriverCapabilities(capabilities)
 
+	// support efs on ecs capabilities
+	capabilities = agent.appendEFSCapabilities(capabilities)
```

Comment:
nit: update comment above capabilities() to include new one

Reply:
to include the new one of what?

Reply:
new capabiliity "ecs.capability.efs"

Reply:
got it, pushed